.github/workflows/turing.yaml

```diff
@@ -27,7 +27,6 @@ on:
       - "docs/**"
       - "engines/pyfunc-ensembler-job/**"
       - "engines/pyfunc-ensembler-service/**"
-      - "sdk/**"
```

Comment:
With the addition of SDK e2e tests, I believe it's probably useful to make the main CI pipeline run (e2e tests) whenever changes to the SDK are made.